### ISSUE TITLE
Need Contribute.md

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act-rules#1]
Need to provide a Contributing document. Keep simple and refer back to wcag-act where possible.

**Comment by moekraft** [comment]
Created initial draft of [contribute.md](https://github.com/w3c/wcag-act-rules/blob/master/contribute.md) .  I have one open question around peer reviews. Please submit for review.

**Comment by maryjom** [comment]
@moekraft I have some thoughts/questions:

- Which group(s) do you need to be active in to contribute?  ACT TF or auto-wcag community group, or neither?
- Contribute to ACT Test Rules and Submit a Pull Request sections: Do we want contributors to create a branch or a fork? Most other W3C groups I work with use only branching. I thought forks can't be seen by others until a pull request is merged. It was also my understanding (maybe incorrect) that forks are easier to get files out of sync and harder to resolve merge issues than branches.  
- Add a bullet - If the contribution is to resolve an issue, add a reference to the issue in the pull request.
- The term 'acceptance criteria' should be a link to the review process section [Acceptance](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/ACT_Review_Process#Acceptance)
- Peer Review section: Should reference the "Peer" section of the review process. I agree there should be labels used on the issues and pull requests opened so we can keep track of where in the process the request is. We should have labels such as draft, ready for review, on current survey (or something that indicates it's under review),  resolving issues, ACT approved, declined, ready for AGWG approval, AGWG reviewing, AGWG approved. I'm not sure how we intend to mark the status of the progression of the rule to make it easy to look at the repository and be able to gather together the completed rules (approved by AGWG, not just the ACT TF, vs. the ones reviewed/approved by the ACT TF waiting for AGWG approval). I know we want to send batches of rules to the AGWG for their review and this may be an easy way to sort on the rules to collect the next batch to send to them. I'm open to other suggestions.
- For the Participate in the ACT TF: I think there needs to be a description (or pointer to one) of what ACT TF does vs. Auto-WCAG Community Group with a description of how to participate in either group or if membership in the Auto-WCAG CG is required to work on/submit rules.
- Editorial: 'github' or 'Github' or 'GitHub' - be consistent, as I see all 3 permutations in this doc.
- Editorial: 'Task Force' not 'Taskforce'
- Editorial: #3 in Submit a Pull Request section: Instead of saying 'reason for the decline' suggest 'reason it was declined'
- Editorial in Code of Conduct: Suggest making this shorter by saying 'This commitment calls for a workplace where participants behave according to the W3C's Code of Ethics and Professional Conduct. (with the link being included in the sentence). That way you could delete the last 2 sentences.

**Comment by moekraft** [comment]
@maryjom 

- You don't have to be active in the task force nor the community group to contribute rules
- You should to create a local copy, fork, and then a branch for the update.
- Added: If the contribution is to resolve an issue, add a reference to the issue in the pull request.
- Linked to Acceptance definition
- Linked to Peer Review section
- Let's discuss labels and process further when reviewing the document
- Added a description of ACT TF and Auto WCAG community group
- It's GitHub
- Changed: 'Taskforce' to 'Task Force' to match rules format however dictionary shows it as a single compound word
Updated: 'reason it was declined'
Simplified text around: Code of Conduct
